lib/tasks/pipeline.js

```diff
@@ -93,11 +92,29 @@ module.exports = Task.extend({
 
     var pluginNameRegex = /^(ember\-cli\-deploy\-)(.*)$/;
     var name            = addon.name.match(pluginNameRegex)[2];
-    if (configuredPluginList && configuredPluginList.indexOf(name) === -1) {
-      return;
-    }
 
-    var deployPlugin = addon.createDeployPlugin({name: name});
+    var configuredNameRegex = /(.*?):(.*)/;
+    var configuredPluginHash = Object.create(null);
+    (deployConfig.plugins || []).forEach(function (pluginName) {
+      var split = pluginName.split(':');
+      if(split[0] === name) {
+        if(split[1]) {
+          configuredPluginHash[pluginName.match(configuredNameRegex)[2]] = split[0];
```

Comment:
Ah, no. `.split(':', 2);` does something else. It limits the elements in the resulting array to 2 but it does not concatenate all others that are left over.
e.g.
```
"redis:foo:bar:doo:bar:doo".split(':', 2)
["redis", "foo"]
```

but my code returns `foo:bar:doo:bar:doo` as the  plugin alias. `split[1]` would in any case only contain `foo`.

As I said, I don't mind. But maybe we should error out if `split(':').length > 2` even if you don't want to support it.